source/fuzz/transformation_add_early_terminator_wrapper.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "source/fuzz/transformation_add_early_terminator_wrapper.h"
+
+#include "source/fuzz/fuzzer_util.h"
+#include "source/util/make_unique.h"
+
+namespace spvtools {
+namespace fuzz {
+
+TransformationAddEarlyTerminatorWrapper::
+    TransformationAddEarlyTerminatorWrapper(
+        const spvtools::fuzz::protobufs::
+            TransformationAddEarlyTerminatorWrapper& message)
+    : message_(message) {}
+
+TransformationAddEarlyTerminatorWrapper::
+    TransformationAddEarlyTerminatorWrapper(uint32_t function_fresh_id,
+                                            uint32_t label_fresh_id,
+                                            uint32_t opcode) {
```

Comment:
It would probably be better to use `SpvOp` here instead of `uint32_t`. The advantage is that we get compiler errors if we pass some invalid value as an argument.

Reply:
Good point - done.